### PR TITLE
Initial Jekyll config

### PR DESCRIPTION
**Pull request by oibind**
SCR-20241119-rlne

### CODE REVIEW

File: js/bootstrap.js
Comment (ohainle) [inline js/bootstrap.js:1]: Either commit `js/***.min.js`, or `js/***.js`, but not both.

File: img/footer.jpg
Comment (ohainle) [inline img/footer.jpg:1]: Seeing multiple copies of everything - assuming no alternatives?

Maybe different development and deployment repos?

Comment (oibind) [reply img/footer.jpg:1]: See: [Add .gitignore and setup Github Pages](https://github.com/qubcaving/qubcaving.github.io/pull/1/commits/73f81364a9ce412b18a049208d8a983df91d7d39)

File: js/bootstrap.js
Comment (oibind) [reply js/bootstrap.js:1]: See: [Remove unneeded duplicates](https://github.com/qubcaving/qubcaving.github.io/pull/1/commits/751f8256cc76773074ccdfe01615199b3609e37f)

File: img/footer.jpg
Comment (oibind) [reply img/footer.jpg:1]: footer.jpg is being used. Just temporarily removed the footer section.

**Review by ohainle — DISMISSED** [review]
Good baseline. Advise looking into GitHub Actions to move as much of the boilerplate (e.g. deps) as possible out of the repo, and into the CI.

**Comment by oibind** [comment]
Create a `dev` branch.

I'll look into GitHub Actions at some point next week.

**Review by ohainle — APPROVED** [review]

**Pull request merged**